packages/near-membrane-base/src/environment.ts

```diff
@@ -296,6 +296,7 @@ export class VirtualEnvironment {
         for (let i = 0, { length } = keys; i < length; i += 1) {
             const key = keys[i];
             bluePointer = this.blueCallableGetPropertyValuePointer(bluePointer, key);
+            // This is where the failure occurs.
```

Comment:
@jdalton @caridy

Reply:
and what exactly is the error? this has been around forever I think

Reply:
> Expected function not to throw, but it threw TypeError: return value not primitive or callable.